src/core/types/time.jl

```diff
@@ -31,6 +38,12 @@ struct TimestepValue{T}
     end
 end
   
+"""
+     TimestepValue
```

Comment:
Sorry didn't catch this till I saw you correct it on the other PR, but this one should be TimestepIndex

Reply:
@ckingdon95 thanks just fixed this, so you're right this encompasses both PRs ... I'll close the docstrings one and wait for tests to pass on this one.  If it looks good now you can approve and we'll be good to go sorry for confusion!